### PR TITLE
feat: add new Regions and support of 'COMMAND+H';fix: fix bug about using sts to log in browser, object's generated download url will be wrong.chore: update dependencies

### PR DESCRIPTION
**Pull request by duan007a**
feat: add new Regions and support of 'COMMAND+H';
fix: fix bug about using sts to log in browser, object's generated download url will be wrong;
chore: update dependencies;

### CODE REVIEW

File: app/main/auth/login.js [hunk 1]
```diff
@@ -193,6 +193,19 @@ angular.module('web')
...
+            var regExp1 = /https:\/\/(\S*)\.aliyuncs\.com/;
+            var res = data.eptpl.match(regExp);
+            if (!res) {
+              res = data.eptpl.match(regExp1);
+            }
+
+            if (res) {
+              data.region = res[1].replace('-internal', '');
+              AuthInfo.save(data);
+            }
```
Comment (aleelock) [inline app/main/auth/login.js]: else 不需要 AuthInfo.save 吗？

File: app/main/auth/login.js [hunk 2]
```diff
@@ -193,6 +193,19 @@ angular.module('web')
 
 
         Auth.login(data).then(function(){
+          if (!data.region && data.eptpl.indexOf('{region}') === -1) {
+            var regExp = /http:\/\/(\S*)\.aliyuncs\.com/;
```
Comment (aleelock) [inline app/main/auth/login.js]: 这2个正则可以合并成一个的

File: node/i18n/ja-JP.js [hunk 3]
```diff
@@ -11,12 +11,16 @@ module.exports = {
...
   'region.oss-cn-beijing': '北京 (中国北部 2)',
   'region.oss-cn-zhangjiakou': '張家口 (中国北部 3)',
+  'region.oss-cn-huhehaote': 'フフホト (中国北部 5)',
   'region.oss-cn-shenzhen': '深セン (中国南部 1)',
   'region.oss-cn-hongkong': '香港',
 
-  'region.oss-ap-southeast-1': 'シンガポール',
-  'region.oss-ap-southeast-2': 'シドニー (オーストラリア)',
-  'region.oss-ap-northeast-1': '東京 (日本)',
+  'region.oss-ap-southeast-1': 'アジア東南 1 (シンガポール)',
```
Comment (aleelock) [inline node/i18n/ja-JP.js]: 和中国和美国的那些风格不一致啊

File: app/main/auth/login.js (hunk 2 above)
Comment (aleelock) [reply app/main/auth/login.js]: /https?:\/\/(\S*)\.aliyuncs\.com/

Comment (duan007a) [reply app/main/auth/login.js]: 已修改

**Pull request merged**